docs/jobs/qgis_installation_finder.md

```diff
@@ -23,11 +23,11 @@ Sample job configuration in your scenario file:
 - name: Find installed QGIS
   uses: qgis-installation-finder
   with:
-    version_priority:
+    - version_priority:
       - "3.36"
-    search_paths:
+    - search_paths:
       - D:\\Applications\\QGIS\\
-    if_not_found: error
+    - if_not_found: error
```

Comment:
Are you sure about this?

Reply:
I'm pretty sure that QDT expects for lists not arrays. It's more an error on the JSON Schemas side.

Reply:
It could be a JSON schema error indeed